core/src/main/java/com/graphhopper/routing/lm/LMPreparationHandler.java

```diff
@@ -93,15 +85,6 @@ public int getLandmarks() {
         return landmarkCount;
     }
 
-    public LMPreparationHandler setDisablingAllowed(boolean disablingAllowed) {
-        this.disablingAllowed = disablingAllowed;
-        return this;
-    }
-
-    public final boolean isDisablingAllowed() {
-        return disablingAllowed || !isEnabled();
-    }
```

Comment:
This was a bit weird. The default for LM disabling allowed was false, but still this returned true whenever there were some LM profiles. -> I changed the default for `lmDisablingAllowed` to true in `RouterConfig` now. Is this ok?

Reply:
Or should we set it to `false` now? Seems a bit more simple to me to have both set to false :)

Reply:
Hm I'll leave it as it is for now we can change the default still later if we think its better.

Reply:
Yes, this could work too. The (hopefully ;)) only requirement for production is that the behaviour of ch.disable=true does not depend on if there are LM profiles or not (as we have clusters with CH and LM but also with CH and without LM). An additional assumption there is that no one uses lm.disable=true and only ch.disable=true/false.

Reply:
Ok I think ch.disable=true does not depend on the existence of LM profiles, but anyway for now it seems easier to not change it :)